internal/plugintypes/timestamp.go

```diff
@@ -12,13 +12,29 @@ type TimestampedBig struct {
 	Value     cciptypes.BigInt `json:"value"`
 }
 
+// TimestampedUnixBig Maps to on-chain struct
+// https://github.com/smartcontractkit/chainlink/blob/37f3132362ec90b0b1c12fb1b69b9c16c46b399d/contracts/src/v0.8/ccip/libraries/Internal.sol#L43-L47
+//
+//nolint:lll //url
+type TimestampedUnixBig struct {
+	Timestamp uint32           `json:"timestamp"`
```

Comment:
What kind of timestamp is this, is this millis, seconds, ?

Reply:
What's used on chain? 

https://github.com/smartcontractkit/chainlink/blob/37f3132362ec90b0b1c12fb1b69b9c16c46b399d/contracts/src/v0.8/ccip/libraries/Internal.sol#L43-L47

Reply:
Check the usage, if its set to block.timestamp then its seconds since the epoch